firefox_puppeteer/__init__.py

```diff
@@ -57,6 +57,12 @@ def prefs(self):
         See the :class:`~api.prefs.Preferences` reference.
         """
 
+    @use_class_as_property('api.appinfo.AppInfo')
+    def appinfo(self):
+        """
+        Provides access to members of the appinfo  api.
```

Comment:
Please also add a .rst file and a reference to the class similar to the other puppeteer properties. So we can extend that later.